docs/store-operations/catalog/graphql-admin/product-custom-fields.mdx

```diff
@@ -0,0 +1,325 @@
+
+## Set product custom fields
+
+The following example sets custom fields for the global store and the specified channel locale. 
+
+<Tabs items={['Request', 'Response']}>
+  <Tab>
+
+    ```graphql filename="Example mutation: Set product custom fields" showLineNumbers copy
+    POST https://api.bigcommerce.com/stores/{{STORE_HASH}}/graphql
+    X-Auth-Token: {{ACCESS_TOKEN}}
+    Content-Type: application/json
+    Accept: application/json
+
+    mutation ($input: SetProductCustomFieldsInformationInput!) {
+      product {
+        setProductCustomFieldsInformation(input: $input) {
+          product {
+            customFields {
+              edges {
+                node {
+                  id
+
+                  # For the global store
+                  name
+                  value
+
+                  # For the channel locale
+                  overrides {
+                    ... on ProductCustomFieldOverridesForChannelLocale {
+                      name
+                      value
+                      isVisible
+                      context {
+                        channelId
+                        locale
+                      }
+                    }
+                  }
+                }
+              }
+            }
+          }
+        }
+      }
+    }
+    ```
+    ```json filename="GraphQL variables" showLineNumbers copy
+    {
+      "input": {
+        "productId": "bc/store/product/111",
+        "data": [
+          {
+            "customFieldId": "bc/store/productCustomField/1",
+
+            // For the global store
+            "name": "Custom Field Name",
+            "value": "Custom Field Value",
+
+            // For the channel locale
+            "overrides": {
+              "channelLocaleContextData": {
+                "context": {
+                  "channelId": "bc/store/channel/1",
+                  "locale": "en-US"
+                },
+                "data": {
+                  "name": "Custom Field Name Override",
+                  "value": "Custom Field Value Override",
+                  "isVisible": true
+                }
+              }
+            }
+          }
+        ]
+      }
+    }
+    ```
+  </Tab>
+  <Tab>
+
+    ```json filename="Example mutation: Set product custom fields" showLineNumbers copy
+    {
+      "data": {
+        "product": {
+          "setProductCustomFieldsInformation": {
+            "product": {
+              "customFields": {
+                "edges": [
+                  {
+                    "node": {
+                      "id": "bc/store/productCustomField/1",
+                    
+                      // For the global store
+                      "name": "Custom Field Name",
+                      "value": "Custom Field Value",
+
+                      "overrides": {
```

Comment:
I guess here the structure should contain connection nodes (`"edges"` and `"node"`) as well, the similar as in the example below

Reply:
Thanks for catching, I just tested / changed the examples